backend/Rhyolite/Backend/App.hs

```diff
@@ -260,12 +260,12 @@ feedPipeline getNextNotification qh r = do
   let qhSaveQuery = QueryHandler $ \q -> do
         atomicModifyIORef' currentQuery $ \old -> (q <> old, ())
         runQueryHandler qh q
-  tid <- forkIO . forever $ do
+  tid <- worker 10000 $ do
```

Comment:
Why add a delay?

Reply:
If it's going to crash every time, then this delay means it only tries to restart a few times per second instead of a few thousand, disk fills up more slowly.  the most convenient way to get the restarting behavior *takes* a delay,  I could choose zero or any other number.

Reply:
Sensible